app/scripts/controllers/bridge-status/validators.test.ts

```diff
@@ -173,56 +210,61 @@ describe('validators', () => {
   describe('bridgeStatusValidator', () => {
     // @ts-expect-error - it.each is a function
     it.each([
+      // {
```

Comment:
Should we remove this instead of commenting out?

Reply:
Whoops good catch, I commented out during development, will put back in

Reply:
Done in [46e7e8d](https://github.com/MetaMask/metamask-extension/pull/29065/commits/46e7e8df781082210d5d59ae084449dfec854f7b)